Backend/app/rider-platform/rider-app/Main/src/API/UI/Ride.hs

```diff
@@ -41,16 +42,22 @@ type API =
                   :<|> "status"
                   :> TokenAuth
                   :> Get '[JSON] DRide.GetRideStatusResp
+                  :<|> "share"
```

Comment:
`status` api is already there for this, not need one more separate api.

Reply:
If it is not needed here, @prashant601 you can shift the definition of this API to `Dashboard` folder(of the same module) and avoid calls to `Domain` folder

Reply:
Done @0utkarsh